src/stac_utils/google.py

```diff
@@ -147,29 +147,53 @@ def create_table_from_dataframe(
     load_data_from_dataframe(client, dataframe, project_name, dataset_name, table_name)
 
 
-@Retry(predicate=if_exception_type(*RETRY_EXCEPTIONS))
-def load_data_from_dataframe(
+def get_table_for_loading(
     client: bigquery.Client,
-    dataframe: Any,
     project_name: str,
     dataset_name: str,
     table_name: str,
 ):
-    """Loads data from the specified dataframe into the specified table in BigQuery"""
     dataset_ref = bigquery.Dataset(project_name + "." + dataset_name)
     table_ref = dataset_ref.table(table_name)
 
     table = client.get_table(table_ref)
+    print("ready to insert rows")
```

Comment:
remove or convert to logging. Also add some descriptive info (table name? row coiunt?)